BZ-25002: feat: add slot and subtext functionality to Toast Component

adarshjuspay opened the pull request:
- addition of a slot for custom content within the toast
- implementation of a subtext option to display secondary text

rs-prasad commented on src/lib/Toast/Toast.svelte:
```diff
@@ -172,20 +189,24 @@
   .error {
     color: var(--toast-error-text, #fff);
     background-color: var(--toast-background-color, #f04438);
+    border-left: var(--toast-error-border-left);
```
Can we add default value for warn/info/error/success, so that we don't require to define css variables.

adarshjuspay replied:
currently border left required only in lighthouse and this component used in other repo and have different usecase

rs-prasad commented on src/lib/Toast/Toast.svelte:
```diff
@@ -120,14 +127,16 @@
     justify-content: var(--toast-justify-content, space-between);
     z-index: var(--toast-z-index, 1000);
-    display: var(--taost-dispay, flex);
+    display: var(--toast-display, flex);
```
You have changed this variable, make sure to update this variable in if used anywhere

rs-prasad commented on src/lib/Toast/Toast.svelte:
```diff
@@ -120,14 +127,16 @@
     height: var(--toast-height, auto);
     border-radius: var(--toast-border-radius, 0px);
     width: var(--toast-width, auto);
-    align-items: var(--toast-align-items, center);
```
Why we have removed this default value?

adarshjuspay replied:
back to default value

adarshjuspay committed:
BZ-25002: feat: add slot and subtext functionality to Toast Component
- addition of a slot for custom content within the toast
- implementation of a subtext option to display secondary text
